application/libraries/Api/Command/V1/SurveyArchive.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace LimeSurvey\Api\Command\V1;
+
+use CHttpSession;
+use Survey;
+use LimeSurvey\Api\Command\{
+    CommandInterface,
+    Request\Request,
+    Response\Response,
+    Response\ResponseFactory,
+    ResponseData\ResponseDataError
+};
+use LimeSurvey\Api\Command\Mixin\Auth\AuthPermissionTrait;
+
+class SurveyArchive implements CommandInterface
+{
+    use AuthPermissionTrait;
+
+    protected CHttpSession $session;
+
+    protected ResponseFactory $responseFactory;
+
+    protected Survey $survey;
+
+    /**
+     * Constructor
+     * @param \CHttpSession $session
+     * @param \LimeSurvey\Api\Command\Response\ResponseFactory $responseFactory
+     * @param \Survey $survey
+     */
+    public function __construct(
+        CHttpSession $session,
+        ResponseFactory $responseFactory,
+        Survey $survey
+    ) {
+        $this->session = $session;
+        $this->responseFactory = $responseFactory;
+        $this->survey = $survey;
+    }
+
+    /**
+     * Processes the request
+     * @param \LimeSurvey\Api\Command\Request\Request $request
+     */
+    public function run(Request $request)
+    {
+        $surveyId = (int)$request->getData('_id');
+        if ($response = $this->ensurePermissions($surveyId)) {
+            return $response;
+        }
+        $survey = Survey::model()->findByPk($surveyId);
+        if (!$survey) {
+            return $this->responseFactory->makeErrorNotFound(
+                (new ResponseDataError(
+                    'SURVEY_NOT_FOUND',
+                    'Survey not found'
+                )
+                )->toArray()
+            );
+        }
+        require_once "application/helpers/admin/import_helper.php";
```

Comment:
This helper should ideally be refactored in to a service and for re-use between the REST API and the core application. I understand we don't have time to do it now. Maybe its worth adding a comment in the code so we are reminded of it?